uoftscrapers/scrapers/exams/utm/__init__.py

```diff
@@ -110,9 +115,34 @@ def get_page_links(endpoint):
                 for li in soup.find('ul', class_='link').find_all('li')]
 
     @staticmethod
-    def parse_course(course_code):
-        # TODO dynamic month/year values
-        month, year, period = 'apr', 2016, 'apr16'
+    def get_period(d):
+        def get_date(month, date, year):
+            months = {
```

Comment:
You could actually just parse it to get the month equivalent:

```python
month = time.strftime('%b', time.strptime('04', '%m'))
```

Only problem is that it gives back `Jun` instead of `June`.